### ISSUE TITLE
M> Establish Official WeChat Group to Promote RChain to Chinese Users

### ISSUE THREAD

**Issue by rjl493456442** [rchain/bounties#257]
In China, WeChat is now one of the most popular social softwares. Chinese users generally through the WeChat group or WeChat official account to understand the information on the blockchain project, team work progress and other information. 
So building an official WeChat group to promote RChain's blockchain idea and blockchain technology is essential.

Taking into account that in China, WeChat is by far the most active social software, with more than **900 million users**. The effect of using Wechat group will be awesome. 

Besides, Wechat group is similar to **Telegram channel**, but for Chinese users, they need tools like VPN to eliminate firewall restrictions, so using telegrams is cumbersome for them, and most users do not know how to use telegram. Therefore Official Wechat Group is really important.

This task includes (i) Set up an official wechat group (ii) Promote rchain's idea, technology and work progress recently.

The first part of this task should be done before the end of February, and the remaining can be done all the time. 

$2000 budget is suitable for the first part of this task, and the remain part should have it own budget each month.

I have set up the Wechat group now. The QRCode is shown below. Welcome to join the RChain Chinese Official Group! All RChain member is strongly recommended to join

![wechatimg13742](https://user-images.githubusercontent.com/5959481/35547374-ab909522-05b3-11e8-9236-3305b9b8f75f.jpeg)

**Comment by linsheng9731** [comment]
Good idea, if we have a WeChat group would be better!

**Comment by zsluedem** [comment]
We can also set up a Weibo account for annoucing things happening about Rchain. It is like a Tweet account ~!

**Comment by zsluedem** [comment]
Actually , I think QQ group is enough for the group chatting .Wechat Group is really hard for a lot of people to communicate.

**Comment by yyh1102** [comment]
Very nice. Most business people especially investors, and coders in China use WeChat, and they are the mainly target users for RChain, so it's a good way to communicate with them.

**Comment by rjl493456442** [comment]
@zsluedem I don't think so. QQ group is limited to user communication, and there is no good content distribution system. In addition, many users are through the WeChat  Friend Circle to understand the blockchain many of the facts, but in qq social software we can not do these.

WeChat group and WeChat Official Account is really valuable!

**Comment by zsluedem** [comment]
@rjl493456442 WeChat Official Account is a good idea!
Just my opinion, Wechat group is always a Chaos once there are more than 100 people(I think).

**Comment by rjl493456442** [comment]
@zsluedem When there is a lot of people, thing always become Chaos. It is same for QQ Group, Telegram channel and else. So administration is important.

**Comment by zsluedem** [comment]
@rjl493456442 Ok~!Just my opinion~! 
But I agree with you on the WeChat Official Account. I think that's more important~

**Comment by yuhuanmocmoc** [comment]
I also think that WeChat is a popular social tool in China, QQ group is too  outdate.

**Comment by w67320177** [comment]
wow ,its so cool, I think rchain will be the next eth!

**Comment by RowlandZhou** [comment]
I can‘t wait to join this group

**Comment by rjl493456442** [comment]
Feel so glad to see there are more than 110 people have joined our Wechat Group. Keep it.

**Comment by AbnerZheng** [comment]
I've joined the group. Hoping the Chinese community become stronger.

**Comment by sportshark** [comment]
@lapin7 @rjl493456442 There is another RChain group of WeChat, which has been running for 2 months and has been maintained free of charge。
![35606020-b98c6910-0687-11e8-84e0-f102447ae1ca](https://user-images.githubusercontent.com/20704995/35660668-5fd37e60-0748-11e8-8c75-31738f235e77.png)

**Comment by zsluedem** [comment]
@sportshark nice work~!

**Comment by sportshark** [comment]
Of course, this group is not bulit by me.
@rjl493456442 @lapin7

**Comment by Ojimadu** [comment]
Is there a need to create multiple instances of RChain WeChat group since there is already a free, two-month old community that is up running?

**Comment by yyh1102** [comment]
@Ojimadu 
I found out that the group from @sportshark is not the official group but a fans group, and only has 48 members. However the group promoted by this issue has already 117 members now. 
The official group may invite the official members of RChain, or the one they propose finally, to be a manager or leader of the group,  not just the proposers among fans simply. 
I don't know which one is worth believing, but i think we should choose the group with most people, just like we should choose the longest chain in the blockchain. : )

**Comment by wiki-yu** [comment]
I have not joined the wechat group previously but only chat in QQ group, which is more active.
